src/timelock.rs

```diff
@@ -0,0 +1,22 @@
+//! Various functions for manipulating Bitcoin timelocks.
+
+use crate::miniscript::limits::LOCKTIME_THRESHOLD;
+
+/// Returns true if `a` and `b` are the same unit i.e., both are block heights or both are UNIX
+/// timestamps. `a` and `b` are nLockTime values.
+pub fn absolute_timelocks_are_same_unit(a: u32, b: u32) -> bool {
+    n_lock_time_is_block_height(a) && n_lock_time_is_block_height(b)
```

Comment:
I wonder if `n_lock_time_is_block_height(a) == n_lock_time_is_block_height(b)` is more readable to other people, or just me.

Reply:
I like it!